pkg/controller/realmadmin/show.go

```diff
@@ -66,7 +68,45 @@ func (c *Controller) HandleShow() http.Handler {
 			return
 		}
 
-		c.renderShow(ctx, w, realm, stats, userStats)
+		// If the user's requested CSV, return it as such.
+		if _, wantCSV := r.URL.Query()["csv"]; wantCSV {
```

Comment:
I really liked the approach you took in https://github.com/google/exposure-notifications-verification-server/pull/920. I think it makes intuitive sense to make the url `.csv` and `.json` and switch on the incoming accept header (which the browser should set).

Reply:
Ok, I'll add support for JSON too, and put control flow through here.